plugins/headless/hu.bme.mit.gamma.headless.server/src/hu/bme/mit/gamma/headless/server/util/IsEclipseProjectFilter.java

```diff
@@ -0,0 +1,28 @@
+package hu.bme.mit.gamma.headless.server.util;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FilenameFilter;
+
+public class IsEclipseProjectFilter implements FileFilter {
```

Comment:
Wouldn't `EclipseProjectFilter` be a better name?

Reply:
fixed in 82e3b4f